File: tools/dbench/answers/python_haiku/s3/e3_validate.py
a = [34, -2, 19, 150, 42, 27]
v = sum(1 for x in a if 0 <= x <= 120)
i = sum(1 for x in a if x < 0 or x > 120)
s = sum(x for x in a if 0 <= x <= 120)
print(v)
print(i)
print(s)

File: tools/dbench/answers/rust_sonnet/s2/e3_validate.rs
fn main() {
    let ages = [34i64, -2, 19, 150, 42, 27];
    let (valid, invalid, sum) = ages.iter().fold((0usize, 0usize, 0i64), |(v, i, s), &a| {
        if a >= 0 && a <= 120 {
            (v + 1, i, s + a)
        } else {
            (v, i + 1, s)
        }
    });
    println!("{}", valid);
    println!("{}", invalid);
    println!("{}", sum);
}
